packages/test-complexdialogs/botfuel-config.js

```diff
@@ -15,5 +15,5 @@
  */
 
 module.exports = {
-  adapter: 'botfuel',
```

Comment:
How could this work before??

Reply:
we use test-config for tests, this one is not used and that's why it was not detected. I just detected it by visual inspection.

Reply:
ok for "It's very risky if Replay cannot identify bots. I think you can store specific headers in fixtures:"

Reply:
In fact it is not possible to set different App Id as environment parameter for each test and use lerna to call test script on each test, since the tests are async and basically lerna will run all the test script first, even before the first tests finished, and so only the last App Id is taken into account. We can not have multiple App Id env parameter in a single process. So I propose to leave it for the moment and solve this later since this is more complicated and need more time to resolve to have a good and elegant solution.